### DISCUSSION TITLE
Mobile roadmap to Android and iOS?

### DISCUSSION THREAD

**Discussion by globalexport** [tauri-apps/cargo-mobile2#178] [Q&A] [answered]
Where can I find a roadmap of the mobile bundling? I would like to know when tauri mobile is ready to be used for Android and iOS.

**Comment by FabianLars** [comment, marked as answer]
The best we can give you right now is "it's ready when it's ready". We're currently focusing on stabilizing it into beta so we can look into handing it off to the external auditors this year.

**Reply by LetrixZ** [reply]
What would be the best alternative right now? Capacitor (if they're comparable)?
